Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -57,6 +58,23 @@ namespace ROS2
             // Found the component!
             return component;
         }
+
+        //! Returns if entity has component of given type
+        //! @param entity pointer to entity
+        //! @param type tpye of the component
+        //! @returns true if entity has component with given type
```

Comment:
Minor adjustment in wording:
First sentence: "Check whether the entity has a component of the given type"

Typo in "tpye"